service-api/docker/web/Dockerfile

```diff
@@ -1,11 +1,19 @@
 FROM nginx:stable-alpine
 
+RUN chown -R nginx:nginx /var/cache/nginx && \
+    chown -R nginx:nginx /var/log/nginx && \
+    chown -R nginx:nginx /etc/nginx/conf.d && \
+    touch /run/nginx.pid && \
+    chown -R nginx:nginx /run/nginx.pid && \
+    chmod -R 777 /etc/nginx/conf.d
```

Comment:
Just wondering why 777 here, cos that would let any user at all write or execute this.....

Reply:
Wondering why all these chowns are necessary because would think the nginx container ought out-of-the box to have handled having its own nginx user, and permissions being correctly set up to run

Reply:
the chown is in response to some of the challenges presented by having run on execution as nginx, which has lower privileges.